packages/llamaindex/package.json

```diff
@@ -33,8 +33,7 @@
     "magic-bytes.js": "^1.10.0"
   },
   "devDependencies": {
-    "@swc/cli": "^0.5.0",
-    "@swc/core": "^1.9.2",
+    "bunchee": "6.4.0",
```

Comment:
AFAIK we're using now bunchee everywhere - can we set the version in the root package.json? (then it's easier to upgrade bunchee)